Documentation/Signoffs/Sign.md

```diff
@@ -57,12 +55,13 @@ a. LED Strips
 > - Each segment is roughly 7 inches (W) x 13 inches (H) 
 > - The following analysis shows how these seven-segments will meet the required constraints of visibility:
 > > - Length of road parallel to main entrance is roughly 100 feet and width is roughly 20 feet [9]
-> > - Maximum viewing angle when entering the lot is 102 feet [9]
-> > - Optimum viewing distance is 120 feet and maximum viewing distance is 525 feet [8]
+> > - Maximum viewing angle when entering the lot is 102 feet [6]
+> > - Optimum viewing distance is 120 feet and maximum viewing distance is 525 feet [5]
 
 - Keiurot LED strips chosen for:
 > - Already waterproof (IP67 rated)
-> - These LED strips claim to be rated for 1000 Lumens each, which meets the constraint of at least 2000 millicandelas at a 60 degree viewing angle or 1.68 Lumens to be seen in the daylight. 
+> - LED strips rated for 1000 Lumens each, a max viewing apex angle of 160 degrees (average max for LED strips) equates to roughly 318.3 Candela. Over both seven-segment displays (16 inches x 13 inches) equate to 2,371.5 Nits (Cd/m^2) which meets the constraint of at least 1000 Nits total.
+Note: This is typically used for LED/LCD array signs which require much more clarity for optimal legibility [8]. 
```

Comment:
8 says nothing about this.

Reply:
I meant to take that out, it was from the previous source, sorry for that mistake